v2/config_env.go

```diff
@@ -12,6 +12,8 @@ type ConfigEnv struct {
 	OutscaleApiEndpoint *string
 	ProfileName         *string
 	Region              *string
+	X509ClientCert      *string
+	X509ClientKey       *string
```

Comment:
This is nice to support (and needed in https://github.com/outscale-dev/osc-bsu-csi-driver/issues/355) but those options need to be passed to client configuration to really use client certificates.